wiki/Tournaments/oSEAT/2/en.md

```diff
@@ -0,0 +1,298 @@
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 6 months of osu!supporter tag, unique profile badge |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 4 months of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 2 months of osu!supporter tag |
+| *4th place* | 1 month of osu!supporter tag |
+| *5th place* | 1 month of osu!supporter tag |
+
+*Note: 5th place will be determined by playoff.*
+
+## Organisation
+
+The osu! South East Tournament 2 is run by various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Management | ![][flag_SG] [phox](/users/772295), ![][flag_ID] [nya10](/users/1372608), ![][flag_TH] [Trigonoculus](/users/7627013) |
+| Map Selectors | ![][flag_SG] [phox](/users/772295), ![][flag_ID] [nya10](/users/1372608) |
+| Designer | ![][flag_TH] [Trigonoculus](/users/7627013) |
+| Streamer | ![][flag_VN] [Hoaq](/users/7696512) |
+| Referees | ![][flag_SG] [phox](/users/772295), ![][flag_SG] [sinn](/users/4262229), ![][flag_ID] [Praza](/users/3184115), ![][flag_ID] [smh](/users/1629553), ![][flag_MY] [Aasheda\_](/users/7614055), ![][flag_MY] [TheSomething](/users/4025232), ![][flag_MY] [Vexine](/users/4042706), ![][flag_MY] [Stupid Idiot](/users/8355574), ![][flag_MY] [Bedwyr Aorta](/users/10875855), ![][flag_MY] [Kumaxione](/users/2582531), ![][flag_TH] [\[Minoriko\]](/users/4958650), ![][flag_VN] [steve\_04\_](/users/10852911), ![][flag_FR] [\[ Mimir \]](/users/7382734), ![][flag_ID] [Houtarou Oreki](/users/4185566) |
+| Commentators | ![][flag_SG] [woahsia](/users/195946), ![][flag_MY] [Jeffrey](/users/4777134), ![][flag_VN] [Hoaq](/users/7696512) |
+| Web Developers | ![][flag_TH] [Trigonoculus](/users/7627013), ![][flag_DE] [Borengar](/users/3312177) |
+| Statistician | ![][flag_VN] [steve\_04\_](/users/10852911) |
+| Wiki | ![][flag_ID] [fajar13k](/users/7100002) |
+
+## Links
+
+- [Discussion thread](/community/forums/topics/819383)
+- [oSEAT Discord server](https://discord.gg/UbwzpG6)
+- [Livestream](https://www.twitch.tv/oseatournament)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vQYmGObPs8N2QaAlJ7K9mrp0EcFJqcEMYXq8ThHX77iGJh2AOhvE6PZsDIuC318gss-Advnd1eFnMbU/pubhtml)**
+
+-----------------------
+
+## Participants
+
+| Seed | Members |
+| :-- | :-- |
+| Top | ![][flag_ID] [LoidKun](/users/6437601), ![][flag_MY] [wuhua](/users/2932510), ![][flag_SG] [GSBlank](/users/2312106), ![][flag_CA] [dunois](/users/4581715), ![][flag_SG] [M4-K1](/users/5210595), ![][flag_MY] [ShaneLiang](/users/6716499), ![][flag_ID] [Skydiver](/users/4750008), ![][flag_SG] [Raindrop](/users/1155871), ![][flag_PH] [MioMilo](/users/2199427), ![][flag_TH] [san-lotso](/users/5296112), ![][flag_SG] [k\_1tty](/users/5407620), ![][flag_TH] [- Phantasma -](/users/1427407), ![][flag_MY] [Tzero](/users/6088976), ![][flag_MY] [Jeffrey](/users/4777134), ![][flag_MY] [TequilaWolf](/users/3633477) |
```

Comment:
tjangjoet missing on top seed

Reply:
The mentioned player is currently resricted, that's why i didn't have any link for it.
But thanks for pointing it out. Will add in any second.